.yarnrc.yml

```diff
@@ -9,5 +9,11 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
     spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
+    spec: "@yarnpkg/plugin-constraints"
+  - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
```

Comment:
Is this plugin still being used? I wasn't able to find any examples

Reply:
Ah, no, it isn't.

Reply:
Removed in [296cced](https://github.com/MetaMask/controllers/pull/831/commits/296cced4e29a3fbd9ae204cb1614037e57865649).